### DISCUSSION TITLE
How to change bullet list icon

### DISCUSSION THREAD

**Discussion by nazmusrangle** [tinymce/tinymce#7766] [General]
Hi, 
I want to change the icon type of the 'ul' elements from circle to '\'. I am trying to update the styles using this
`
tinymce.init({
  selector: 'textarea#format-custom',
  height: 500,
  plugins: 'link code lists advlist',  
  content_style: '',
  menubar: false,  
  toolbar: 'styleselect | bold italic | link code | bullist',  
  style_formats: [
  {title: 'Headers', 
  items: [  { title: 'Large Heading', block: 'h1', attributes: {class: ''}},
    { title: 'Medium Heading', block: 'h2', attributes: {class: ''}},
    { title: 'Small Heading', block: 'h3', attributes: {class: ''}},
    { title: 'Editorial', block: 'p', attributes: {class: 'editorial'}},
    { title: 'Large Body', block: 'p', attributes: {class: 'body-large'}},
    { title: 'Medium Body', block: 'p', attributes: {class: ''}},
    { title: 'Small Body', block: 'p', attributes: {class: 'body-small'}},
  ]}, 
   {title: 'Bullist', 
   items: [ {
         title: 'Custom Bullet',
             selector: 'ul', 
             classes: 'custom'
         }]}
  ],
  content_style: '.body-small { color: red; }',
  content_style: '.custom { list-style: '\' }'
});
`
Can anyone tell me what I am doing wrong?

**Comment by TheSpyder** [comment]
Because the configuration is a JavaScript string the escape character `\` has special meaning. As a result it needs to be escaped for CSS _and_ escaped for JavaScript. Try this:
`content_style: '.custom { list-style: "\\\\" }'`

Example fiddle:
https://fiddle.tiny.cloud/rbiaab

Alternatively use an external stylesheet and link to it.

**Reply by nazmusrangle** [reply]
Thanks a lot. It worked.

**Comment by ranwenqi** [comment]
How to update in real time if there are changes?
